scripts/jenkins/cloud/manual/configure-deployer-container.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Create deployer container
+  hosts: ci-deployer
+  connection: docker
+  remote_user: root
+  gather_facts: False
+  tasks:
+    - name: Read cloud config files
+      set_fact:
+        clouds: "{{ lookup('file', item) | from_yaml | combine(clouds | default({}), recursive=True) }}"
+      with_fileglob:
+        - "{{ lookup('env', 'HOME') + '/.config/openstack/*' }}"
+
+    - name: Replace cacert paths
+      set_fact:
+        clouds: "{{ clouds | combine(cacert, recursive=True) }}"
+      vars:
+        cacert: "{ 'clouds': { '{{ item }}': { 'cacert': '/usr/share/pki/trust/anchors/SUSE_Trust_Root.crt.pem' } } }"
+      with_items: |
+        [ {% for cloud in clouds['clouds'] %} '{{ cloud }}', {% endfor %} ]
+
+    - name: Create config directory
+      file:
+        state: directory
+        dest: "/root/.config/openstack"
```

Comment:
again, using hardcoded root seems a bit bruteforce. Any way to make this more flexible and maybe use 'root' as default. Don't know if it matters.

Reply:
I wonder if the docker image has a sles user in it. If not, we can always create an unprivileged user in the Dockerfile and run everything as that user. A container is a very constrained environment, so I wouldn't be looking for the flexibility to run as any random user. You get what the image supplies and you work with that. With that said, running as root is always a questionable thing to do so creating an unprivileged user account and assuming that everything runs as that user makes sense.

Reply:
The image doesn't come with any users. Root is the only one. We could of course create an unprivileged user in the Dockerfile. What would the advantages be give that the user has complete control over the image anyhow?

Reply:
What about replacing "/root/" with the equivalent of $HOME
you can also pus the directory to the top of playbook as a var

vars: 
   user_home_dir: "/root"

That way it can be overriden on the command call to the playbook with command form: 
"ansible-playbook configure-deployer-container.yml -e "user_home_dir="/someotheruserhome"

Reply:
Yes, that sounds good.